low-level/sensors/impl/generic/raid.py

```diff
@@ -390,8 +407,14 @@ def _process_missing_md_devices(self, md_device_list, drive_dict):
         for line in raid_conf_data:
             try:
                 raid_conf_field = line.split(" ")
-                if "md" in raid_conf_field[1]:
-                    conf_device_list.append(raid_conf_field[1])
+                if "#" not in raid_conf_field[0] and "ARRAY" in raid_conf_field[0] and \
```

Comment:
Can you add sample of lines you are trying to parse here, so parsing code becomes more clear